Turn/src/Game.cpp

```diff
@@ -274,8 +274,10 @@ void Game::Battle(){
         _Player->DisplayHUD(_Enemy);
         _Enemy->DisplayHUD();
 
+		int damagePlayer = _Player->Attack();
         // Player's turn to attack Enemy.
-        _Enemy->TakeDamage(_Player->Attack());
+
+		_Enemy->TakeDamage(damagePlayer);
```

Comment:
Bug occurs here, when `damagePlayer = 2`, 2 points are added to enemy's health (which makes it look like it's healing)